src/Features/CSharp/Portable/Completion/KeywordRecommenders/LoadKeywordRecommender.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
+{
+    internal class LoadKeywordRecommender : AbstractSyntacticSingleKeywordRecommender
+    {
+        public LoadKeywordRecommender()
+            : base(SyntaxKind.LoadKeyword, isValidInPreprocessorContext: true)
+        {
+        }
+
+        protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
+        {
+            var syntaxTree = context.SyntaxTree;
+            return
+                syntaxTree.IsInteractiveOrScript() &&
+                syntaxTree.IsBeforeFirstToken(position, cancellationToken) &&
+                context.IsPreProcessorKeywordContext;
```

Comment:
I don't think it should be removed.  The check you're talking about (I'm assuming [this one](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Features/Completion/KeywordRecommenders/AbstractSyntacticSingleKeywordRecommender.cs,2066b877997443db,references)) will only skip `IsValidContext()` if we're in a pre-processor context _and_ `_isValidInPreProcessorDirectiveContext` is false, but it's presumably possible that `_isValidIn...` could be true but the context could be false, like in the middle of a regular C# file, and in that case we still need this context check.